src/group/managed_group/mod.rs

```diff
@@ -898,7 +898,7 @@ impl<'a> ManagedGroup<'a> {
                     if let Some(queued_proposal) = pending_proposals_queue.get(proposal_reference) {
                         self.send_proposal_event(
                             queued_proposal.proposal(),
-                            sender,
+                            &queued_proposal.sender().to_leaf_index(),
```

Comment:
This shouldn't borrow. But feel free to do in a follow up.

Reply:
Agreed, it's a bit inconsistent. I can make `.to_leaf_index()` return a `&LeafIndex` and that would be my preferred way. This will lead to some more code changes elsewhere. Let me know if you're fine with that.

Alternatively I could just change the function signature of `send_proposal_event()` to accept a `LeafIndex` and copy the value. Less preferred, but also less code changes.

Reply:
Well, `LeafIndex` really is a `u32`, which should be passed by value. If the compiler actually uses a reference here, it's probably more expensive than passing the value. So I'd recommend doing the second option.